### DISCUSSION TITLE
Github foundation, outdated question about roles?

### DISCUSSION THREAD

**Discussion by ChrisHLC** [v-fidelusaleksander/ghcertified#232] [Study Together!]
![2024-05-23 19_55_32-Window](https://github.com/FidelusAleksander/ghcertified/assets/33631473/1293afd4-d240-480d-9709-7c51e8ac457b)

Hi, I'm trying to find some documentation about this topic but can't find any of those roles, unfortunately, the [link ](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/permission-levels-for-a-personal-account-repository)in the question doesn't help.

I found this [link](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization#access-requirements-for-security-features) and it looks like the roles have been updated.

**Comment by ChrisHLC** [comment]
My bad this link shows the roles in the question
https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization

**Reply by v-fidelusaleksander** [reply]
@ChrisHLC you raised a good point - the documentation link was wrong. I've updated the question with correct link in https://github.com/FidelusAleksander/ghcertified/pull/233

Thanks for raising this :+1:  Next time you see something similarly wrong - please raise an Issue in the repository :smile:
